misc/helm-charts/operator/values.yaml

```diff
@@ -27,6 +27,9 @@ operator:
     # -- Whether to enable environmentd rbac checks
     # TODO: this is not yet supported in the helm chart
     authentication: false
+    # -- Whether to enable netowrk policies
+    # -- Network policies don't yet work in self-managed
+    disable_database_network_policies: true
```

Comment:
Nitpick, comment spelling.

Nitpick, negative bools are often confusing, especially if we are inverting that for the actual parameter passed to environmentd.